packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -221,7 +221,34 @@ public function createBasicConfig(): void
                 ],
                 'Examples' => [
                     'screenshots' => [
-                        [
+                        "_default"=> [
+                            ["action"=> "resizeWindow", "width"=> 640,"height"=> 640],
+                            ["action"=> "setScreenshotsDocumentationPath", "path"=> "Documentation"],
+                            ["action"=> "setScreenshotsImagePath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setScreenshotsRstPath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setCodeSnippetsSourcePath","path"=> "typo3conf/ext/examples"],
+                            ["action"=> "setCodeSnippetsTargetPath", "path"=> "CodeSnippets"]
+                        ],
+                        "frontend" => [
+                            ["include" => "_default"],
+                            [
+                                "action" => "resizeWindow",
+                                "width" => 480,
+                                "height" => 640
+                            ],
+                            [
+                                "action" => "amOnPage",
+                                "page" => "/?id=1"
+                            ],
+                            ["action" => "wait", "timeout" => 1],
+                            [
+                                "action" => "makeScreenshotOfElement",
```

Comment:
Prefer `makeScreenshotOfWindow` over `makeScreenshotOfElement` if no selector is specified ..